util/env_posix.cc

```diff
@@ -675,6 +675,10 @@ class PosixEnv : public Env {
 
   virtual void SleepForMicroseconds(int micros) override { usleep(micros); }
 
+  virtual void SleepForNanoseconds(int nanos) override {
```

Comment:
Or we can call SleepForMicroseconds and not adding the SleepForNanoseconds, since sleep is not gonna provide nano-level granularity.